draft-ietf-tls-tls13.md

```diff
@@ -3872,91 +3828,35 @@ and renegotiation_info {{RFC5746}}.
 
 ## Cipher Suites
 
-A cipher suite defines a cipher specification supported in TLS and negotiated
-via hello messages in the TLS handshake.
-Cipher suite names follow a general naming convention composed of a series
-of component algorithm names separated by underscores:
+A symmetric cipher suite defines the pair of the AEAD cipher and hash
+function to be used with HKDF. Cipher suites follow the naming convention:
+Cipher suite names follow the naming convention:
 
 ~~~
-   CipherSuite TLS_KEA_AUTH_WITH_CIPHER_HASH = VALUE;
+   CipherSuite TLS13_CIPHER_HASH = VALUE;
 ~~~
 
 | Component | Contents |
 |:----------|:---------|
-| TLS       | The string "TLS" |
-| KEA       | The key exchange algorithm (e.g. ECDHE, DHE) |
-| AUTH      | The authentication algorithm (e.g. certificates, PSK) |
-| WITH      | The string "WITH" |
+| TLS13     | The string "TLS13" |
```

Comment:
Do we expect these to be used for TLS 1.4 as well? Wondering if we should pick some more generic name like `TLS_AEAD_AES_128_GCM_SHA256` or maybe just `TLS_AES_128_GCM_SHA256`.